tests/suites/test_suite_ssl.function

```diff
@@ -177,22 +181,29 @@ int mbedtls_test_message_queue_setup( mbedtls_test_message_queue *queue,
 
 void mbedtls_test_message_queue_free( mbedtls_test_message_queue *queue )
 {
+    if( queue == NULL )
+        return;
```

Comment:
This should be added in "Add a message metadata queue in ssl tests", not "Add a message-based socket mock connection to the ssl tests", along with a test.